lib_eio_luv/eio_luv.ml

```diff
@@ -455,6 +470,28 @@ module Low_level = struct
         t.fd <- `Closed;
         Eio.Switch.remove_hook t.release_hook;
         fd
+
+    let kind_map =
+      (* we only need 12, but 13 maps directly *)
+      let km = Array.init 13 (fun _ -> `Unknown) in
+      Array.set km 0o1  `Fifo;
```

Comment:
Where do these numbers come from?

Reply:
I'll clarify in a comment, but I was mostly trying to avoid having a `discovery.ml` for `S_IF*`, the values are actually standarized and I've checked linux/OpenBSD/FreeBSD/OS X. The array of 13 is that we can store say `S_IFSOCK (0o140000) at km.(0o14) instead of 0o13`.

Reply:
I've removed this in favor of an association list